lib/ecmascript.ts

```diff
@@ -1197,9 +1367,11 @@ export function ToTemporalMonthDay(item, options = ObjectCreate(null)) {
   }
 
   ToTemporalOverflow(options); // validate and ignore
-  let { month, day, referenceISOYear, calendar } = ParseTemporalMonthDayString(ToString(item));
-  if (calendar === undefined) calendar = GetISO8601Calendar();
-  calendar = ToTemporalCalendar(calendar);
+  let { month, day, referenceISOYear, calendar: maybeStringCalendar } = ParseTemporalMonthDayString(ToString(item));
+  // TODO: should this be a ternary?
```

Comment:
as

`const calendar = maybeStringCalendar ? ToTemporalCalendar(maybeStringCalendar) : GetISO8601Calendar();`

Seems reasonable to me, though should empty string be treated the same as undefined? Or should it be an explicit undefined check?

Reply:
For now I'm generally leaving these cases as-is to align with proposal-temporal. I'd be open to cleaning them up when there's less change coming from the other repo, maybe in a few months? Sound OK?